config/configmap/inferenceservice.yaml

```diff
@@ -537,22 +537,41 @@ data:
            "imagePullSecrets": ["docker-secret"]
        }
      
-     # ====================================== DEPLOYMENT CONFIGURATION ======================================
-     # Example
-     deploy: |-
-       {
-         "defaultDeploymentMode": "Serverless"
-       }
-     deploy: |-
-       {
-         # defaultDeploymentMode specifies the default deployment mode of the kserve. The supported values are
-         # Serverless, RawDeployment and ModelMesh. Users can override the deployment mode at service level
-         # by adding the annotation serving.kserve.io/deploymentMode.For more info on deployment mode visit
-         # Serverless https://kserve.github.io/website/master/admin/serverless/serverless/
-         # RawDeployment https://kserve.github.io/website/master/admin/kubernetes_deployment/
-         # ModelMesh https://kserve.github.io/website/master/admin/modelmesh/
-         "defaultDeploymentMode": "Serverless"
-       }
+    # ====================================== DEPLOYMENT CONFIGURATION ======================================
+    # Example
+    deploy: |-
+      {
+        "defaultDeploymentMode": "RawDeployment",
+        "rawDeploymentRolloutStrategy": {
```

Comment:
We already changed rawDeployment in https://github.com/kserve/kserve/pull/4608/files, let's rename this to `deploymentRolloutStrategy` ?